daemon/containerd/image_inspect.go

```diff
@@ -101,9 +102,10 @@ func (i *ImageService) ImageInspect(ctx context.Context, refOrID string, _ backe
 	}
 
 	return &imagetypes.InspectResponse{
-		ID:            img.ImageID(),
-		RepoTags:      repoTags,
-		RepoDigests:   repoDigests,
+		ID:       img.ImageID(),
+		RepoTags: repoTags,
+		// TODO: Use maps.Keys from stdlib once we move the module to go >= 1.23
+		RepoDigests:   maps.Keys(repoDigests),
```

Comment:
It seems the goal is to deduplicate `repoDigests`. FWIW, we've `sliceutil.Dedup` for that 😉